### PR TITLE
Drippin heretic costume

### PR DESCRIPTION
**Pull request by airlines7**
Originally from pull #18832 this is just the robes part of it.

# Description
### TL;DR
Adds a "fashionable robe" to the costume autodrobe as contraband, the costume version of the ominous armor.

### The drippy robe
After all these years the heretic drip now for the common man. I've made sure that it does not decrease damage as it is just a "fashionable" hood and robe. Has some silly flavor text about a massacre at a Nanotrasen fashion show.
Video proof  https://user-images.githubusercontent.com/98609667/236930600-580c9b00-b6a2-4f16-9b83-41228e526487.mp4

image

image

image

## Why?
That robe is way to drippin for the clown or mime to not use when pranking sec, a chaplain could also wear it for much more occult religions. The Sprite is just too damn cool for only the heretic to have.

# Changelog

:cl:  Airlines7
rscadd: Heretic costumes for the comedic clown and cynical mime.
/:cl:

### CODE REVIEW

File: code/modules/clothing/suits/miscellaneous.dm [hunk 1]
```diff
@@ -784,3 +784,19 @@
...
+/obj/item/clothing/head/hooded/cult_hoodie_toy/eldritch_toy
+	name = "fashionable hood"
+	desc = "A stylish and trendy hood perfectly pairing with the robes it comes with. Strange eyes line the inside"
+	icon_state = "eldritch"
+	item_state = "eldritch"
+	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE|HIDEHAIR|HIDEFACIALHAIR
+
+/obj/item/clothing/suit/hooded/cultrobes/eldritch_toy
+	name = "fashionable robe"
+	desc = "A stylish and trendy robe straight from this year's Nanotrasen fashion show. Though you can't rememeber seeing this entry on the catwalk."
```
Comment (Xoxeyos) [inline code/modules/clothing/suits/miscellaneous.dm]: Grammar Check: Stylish and trendy mean the same thing, put a comment after "hood", if you have a descriptor beforehand. Erase the first "with"; "pairing the robes it comes with" sounds better than "pairing with the robes it comes with.". Rememeber should be 'Remember'. 'Though' in the robe section is a bit obtuse, you can just have it state "You can't remember seeing this entry on the catwalk.".

Comment (Xoxeyos) [reply code/modules/clothing/suits/miscellaneous.dm]: >Nerd emoji nerd emoji nerd emoji.

Comment (airlines7) [reply code/modules/clothing/suits/miscellaneous.dm]: nerd emoji-ing himself

**Comment by airlines7** [comment]
Thanks for that grammar check.

**Comment by Xoxeyos** [comment]
> Thanks for that grammar check.

You're welcome.

**Review by ynot01 — APPROVED** [review]
